src/components/user-actions/index.tsx

```diff
@@ -62,7 +61,7 @@ export class UserActions extends React.Component<Properties, State> {
       <>
         <div className='user-actions'>
           <button className='user-actions__icon-button'>
-            <IconMessageSquare2 isFilled={this.props.isConversationListOpen} />
+            <IconMessageSquare2 isFilled />
```

Comment:
Better solution here could have been to just remove this component as it's never rendered any more.